src/App/Components/ArticleSocialSharing/ArticleSocialSharing.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import axios from '../../../configs/axios';
+
+const ArticleSocialSharing = (props) => {
+  const { slug, title } = props;
+  const articleLink = `https://authors-haven-staging.herokuapp.com/articles/${slug}`;
+  const handleShare = (social) => {
+    switch (social) {
+      case 'twitter':
+        axios.post(`api/articles/${slug}/share/twitter`);
+        break;
+      case 'linkedin':
+        axios.post(`api/articles/${slug}/share/linkedin`);
+        break;
+      case 'facebook':
+        // eslint-disable-next-line no-undef
+        FB.ui({
+          method: 'send',
+          link: articleLink
+        });
+        axios.post(`api/articles/${slug}/share/facebook`);
+        break;
+      case 'email':
+        axios.post(`api/articles/${slug}/share/gmail`);
+        break;
+      default:
+    }
+  };
+  return (
+    <div className="col-xl-3 col-lg-3 col-md-12 col-sm-12 col-12">
+      <div className="social-share__container">
+        <button type="button" className="btn twitter-share">
+          <a
+            id="twitter-share"
+            href={`https://twitter.com/intent/tweet?text=${title} - ${articleLink}`}
```

Comment:
I remember you said that you don't know why it is opening two tabs but this is the reason: you have an `a` tag which has a `path` and an `onclick` event listener. choose between one of them and if you opt for the `onclick` it is supposed to be a `button`

Reply:
alright thank you @alainburindi